pubspec.lock

```diff
@@ -278,18 +278,18 @@ packages:
     dependency: "direct main"
     description:
       name: drift
-      sha256: b584ddeb2b74436735dd2cf746d2d021e19a9a6770f409212fd5cbc2814ada85
+      sha256: e60c715f045dd33624fc533efb0075e057debec9f39e83843e518f488a0e21fb
       url: "https://pub.dev"
     source: hosted
-    version: "2.26.1"
+    version: "2.27.0"
```

Comment:
I'm seeing some deprecation warnings from this upgrade:

https://github.com/simolus3/drift/commit/4116b95ef9fcfcda7092af8c633ccec05ec97d5b

```
$ flutter analyze
Analyzing zulip-flutter...

   info • 'validateDropped' is deprecated and shouldn't be used. Use field in ValidationOptions instead • test/model/database_test.dart:204:28 • deprecated_member_use
   info • 'validateDropped' is deprecated and shouldn't be used. Use field in ValidationOptions instead • test/model/database_test.dart:214:59 • deprecated_member_use

2 issues found. (ran in 5.2s)
```